funnel/views/project.py

```diff
@@ -572,6 +573,9 @@ def register(self):
                 rsvp.rsvp_yes()
                 db.session.commit()
                 flash(_("You have successfully registered"), 'success')
+                user_registered_for_project.send(
+                    project=self.obj, user=current_auth.user
+                )
```

Comment:
Signals require a positional parameter for the sender. This call should be `signal_name.send(self.obj, user=current_auth.user)`